### PR TITLE
Laravel horizon

### PR DESCRIPTION
**Pull request by sbsmith86**
:chompy: 's first PR

This PR installs [Laravel Horizon](https://laravel.com/docs/5.6/horizon) to help manage queue jobs.

### CODE REVIEW

**Comment by weerd** [comment]
Oh sweet! You're using Horizon! 🌅 🎉 Noice!!

**Comment by sbsmith86** [comment]
it's real cool

File: app/Providers/AppServiceProvider.php [hunk 1]
```diff
@@ -15,6 +15,11 @@ class AppServiceProvider extends ServiceProvider
     public function boot()
     {
         Schema::defaultStringLength(191);
+
+        // Horizon Dashboard Authentication.
+        \Horizon::auth(function ($request) {
+            return true;
```
Comment (DFurnes) [inline app/Providers/AppServiceProvider.php]: Should this return `auth()->user()->role === 'admin'`?

Comment (sbsmith86) [reply app/Providers/AppServiceProvider.php]: oh yea that make sense.

**Pull request merged**